backend/infrahub/events/branch_action.py

```diff
@@ -18,9 +22,9 @@ def get_name(self) -> str:
 
     def get_resource(self) -> dict[str, str]:
         return {
-            "prefect.resource.id": f"infrahub.branch.{self.branch}",
+            "prefect.resource.id": f"infrahub.branch.{self.branch_name}",
```

Comment:
Wondering a bit how the `prefect.resource.id` will be used. If a user deletes a branch within Infrahub and then creates a new one with the same name do we still want it to use the same `prefect.resource.id` ?